DigThemGraves/Assets/!Shortcuts/MoneyScipt.asset

```diff
@@ -0,0 +1,15 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: f9721878bc103c14996724acdc779861, type: 3}
+  m_Name: MoneyScipt
```

Comment:
Literówka?

Reply:
Przenieś proszę !Shortcuts wewnątrz do pluginów. Pewnie trzeba będzie zmodyfikować jakąś zmienną w paczce `RealShortcuts` ale nie powinno być to zbyt ciężkie.

Reply:
tzn. !Shortcuts to jest folder stworzony przeze mnie, żeby był zawsze pierwszy w hierarchii xd

Reply:
fair enough